### PR TITLE
make it all run again

### PR DESCRIPTION
**Pull request by jspaaks**
- Copied a bunch of config from earlier projects
- repaired stuff to make it run again

branch fails because of https://github.com/fair-software/howfairis-github-action/issues/41

but a local `howfairis https://github.com/sequgen/sequgen --branch run-again` gets it right

### CODE REVIEW

**Review by eriktks — APPROVED** [review]
Tested this PR because currently the tests on main fails and this PR solves that. Inspected the file changes and they look fine. Great to have dev instructions. isort suggested import swaps on four files: docs/conf.py setup.py tests/test_five_models.py tests/test_mvp.py. All else is ok.

**Comment by eriktks** [comment]
It looks like the checks fail because howfairis looks at the live version rather than the branch. The [report](https://github.com/sequgen/sequgen/pull/29/checks?check_run_id=2182745053) states:

```
Checking compliance with fair-software.eu...
url: https://github.com/sequgen/sequgen
(1/5) repository
```

(no mention of branch) while if I run howfairis from the command line with option `-b`, I get:

```
Checking compliance with fair-software.eu...
url: https://github.com/sequgen/sequgen
branch: run-again
(1/5) repository
```

(branch mentioned on line 3).

**Comment by jspaaks** [comment]
> isort suggested import swaps on four files: docs/conf.py setup.py tests/test_five_models.py tests/test_mvp.py. All else is ok.

my `isort` (and `prospector`) is quiet. What command did you run to get this?

**Comment by sverhoeven** [comment]
> > isort suggested import swaps on four files: docs/conf.py setup.py tests/test_five_models.py tests/test_mvp.py. All else is ok.
> 
> my `isort` (and `prospector`) is quiet. What command did you run to get this?

You get those warnings if you run isort outside the sequgen/ dir (one with `__init__.py`), see README.dev.md for commands that only check sequgen dir.

**Comment by eriktks** [comment]
Here are the commands which I used and their output. Indeed, if I do `isort --recursive --check-only sequgen` the command remains silent. Sorry, I had not used the command from `README.dev.md`.

```
$ git checkout remotes/origin/run-again
Note: checking out 'remotes/origin/run-again'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at c9d4e38 added venv permutations to gitignore

$ isort
--- /home/erikt/software/sequgen/setup.py:before	2021-03-24 15:58:40.874017
+++ /home/erikt/software/sequgen/setup.py:after	2021-03-24 15:58:45.023102
@@ -1,8 +1,8 @@
 #!/usr/bin/env python
 import os
+from setuptools import setup
 from sequgen.__version__ import __version__
 
-from setuptools import setup
 
 here = os.path.abspath(os.path.dirname(__file__))
 
Apply suggested changes to '/home/erikt/software/sequgen/setup.py' [y/n/q]? n
--- /home/erikt/software/sequgen/tests/test_five_models.py:before	2021-03-24 15:58:40.874017
+++ /home/erikt/software/sequgen/tests/test_five_models.py:after	2021-03-24 15:58:53.234710
@@ -1,12 +1,12 @@
+import numpy
 from matplotlib import pyplot as plt
-import numpy
+from sequgen.deterministic.constant import constant as signal3
+from sequgen.deterministic.normal_peak import normal_peak as signal4
 from sequgen.deterministic.sine import sine as signal1
 from sequgen.deterministic.triangular_peak import triangular_peak as signal2
-from sequgen.deterministic.constant import constant as signal3
-from sequgen.deterministic.normal_peak import normal_peak as signal4
+from sequgen.dimension import Dimension
+from sequgen.parameter_space import ParameterSpace
 from sequgen.stochastic.gaussian import gaussian as noise1
-from sequgen.parameter_space import ParameterSpace
-from sequgen.dimension import Dimension
 
 
 def test_five_models():
Apply suggested changes to '/home/erikt/software/sequgen/tests/test_five_models.py' [y/n/q]? n
--- /home/erikt/software/sequgen/tests/test_mvp.py:before	2021-03-24 15:58:40.874017
+++ /home/erikt/software/sequgen/tests/test_mvp.py:after	2021-03-24 15:58:55.281802
@@ -1,8 +1,8 @@
+import numpy
 from matplotlib import pyplot as plt
-import numpy
 from sequgen.deterministic.triangular_peak import triangular_peak
+from sequgen.dimension import Dimension
 from sequgen.parameter_space import ParameterSpace
-from sequgen.dimension import Dimension
 
 
 def test_mvp():
Apply suggested changes to '/home/erikt/software/sequgen/tests/test_mvp.py' [y/n/q]? n
--- /home/erikt/software/sequgen/docs/conf.py:before	2021-03-24 15:57:34.773991
+++ /home/erikt/software/sequgen/docs/conf.py:after	2021-03-24 15:58:57.485584
@@ -18,8 +18,8 @@
 #
 import os
 import sys
-
 import sequgen
+
 
 here = os.path.dirname(__file__)
 sys.path.insert(0, os.path.abspath(os.path.join(here, '..')))
Apply suggested changes to '/home/erikt/software/sequgen/docs/conf.py' [y/n/q]? n
Skipped 1 files
```

File: README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Install
+
+``` {.sourceCode .console}
+pip3 install sequgen
+```
+
+## Contributing
+
+For developer documentation, go [here](README.dev.md).
```
Comment (eriktks) [inline README.md:34]:
```suggestion
For developer documentation, go to the [developer's README](README.dev.md).
```

**Comment by jspaaks** [comment]
>  Sorry, I had not used the command from README.dev.md.

No worries, I just wanted to check if I was perhaps losing it

**Review by sverhoeven — APPROVED** [review]
Better in many ways.

* Pre-commit script works
* tests get stuck, but will get fixed in #36 
* fair software ci workflow works as expected
* bump2version works
* README renders
* coverage/README unable to check listing due to stuck tests
* grr empty docstrings, fixed in #39
* bad practice getting version in setup.py, see #46
* yeah for consistent quotes

**Pull request merged**